### ISSUE TITLE
[Accessibility]: The ARIA attributes "aria-labelledby" are not valid bubble chart

### ISSUE THREAD

**Issue by gssspillai** [carbon-design-system/carbon-charts#1990]
### Application/Team

Cloud pak for data

### Environment

Browser

### What happened? What did you expect to happen instead?

Accessablity issues in bubble chart 'legent' option. 

![Image](https://github.com/user-attachments/assets/c9c95ce7-6f21-4090-bda2-68e58a926afb)

![Image](https://github.com/user-attachments/assets/f0e87b7f-3bb1-4255-9d1f-47073137bb5a)

### What WCAG 2.1 checkpoint does the issue violate?

The ARIA attributes "aria-labelledby" are not valid for the element <div> with implicit ARIA role "generic"

### Version

    "@carbon/charts": "1.13.7",
    "@carbon/charts-react": "^1.22.0",

### Data & options used

```js
'legend': {
      'additionalItems': [
        {
          'type': 'radius',
          'name': 'Alerts'
        }
      ]
    },
```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P2

**Comment by gssspillai** [comment]
@theiliad, can you please suggest any solutions here?
